core/src/lib.rs

```diff
@@ -291,6 +292,45 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    pub fn as_edn_value(&self) -> edn::Value {
```

Comment:
Rust generally uses `as` for cheap casts (views) and `to` for expensive conversions.  See https://aturon.github.io/style/naming/conversions.html.